monitoring/google/cloud/monitoring/query.py

```diff
@@ -314,6 +316,20 @@ def select_metrics(self, *args, **kwargs):
 
             metric.label.<label> = ends_with("<value>")
 
+        As a special case, ``"metric_type"`` is treated as a special
+        pseudo-label corresponding to the filter object ``metric.type``.
+        For example, ``metric_type=<value>`` generates::
```

Comment:
Sorry I forgot to mention this, can you change to the explicit syntax:

```
        For example, ``metric_type=<value>`` generates::

        .. code-block:: python

           metric.type = "<value>"
```

I am currently at the very beginning of a project to make sure all of our snippets get tested with doctest and the explicit blocks are part of the project.